builder/bundleset.go

```diff
@@ -221,6 +221,49 @@ func validateBundle(b *bundle) error {
 	return nil
 }
 
+// parsePundleFile parses a file at filename containing a flat list of package
+// names to be converted to one-package bundles (pundles). Returns a list of
+// pointers to bundle objects created from this list.
+func parsePackageBundleFile(filename string) (map[string]bool, error) {
```

Comment:
Comment and function do not agree, this returns a map of strings, not a list of pointers.

Reply:
Fixed.